src/applications/post-911-gib-status/containers/IntroPageSummary.jsx

```diff
@@ -24,131 +24,147 @@ export default function IntroPageSummary() {
         <strong>Note:</strong> This tool is available Sunday through Friday,
         6:00 a.m. to 10:00 p.m. ET, and Saturday 6:00 a.m. to 7:00 p.m. ET.
       </p>
-      <h2 itemProp="name">Am I eligible to use this tool?</h2>
       <div
-        itemProp="acceptedAnswer"
+        itemProp="mainEntity"
```

Comment:
Same thing here :)

Reply:
I also added a wrapping `<div>` here that wrapped both the question and the answer, otherwise it wasn't picking up the questions correctly.